lib/ansible/module_utils/crypto.py

```diff
@@ -94,12 +94,19 @@ def get_fingerprint(path, passphrase=None):
         return None
 
 
-def load_privatekey(path, passphrase=None, check_passphrase=True, backend='pyopenssl'):
-    """Load the specified OpenSSL private key."""
+def load_privatekey(path, passphrase=None, check_passphrase=True, path_content=None, backend='pyopenssl'):
+    """Load the specified OpenSSL private key.
+
+    The content can also be specified via path_content; in that case,
```

Comment:
`content` rather than `path_content` would make more sense to me as an argument name.

Reply:
Fixed.